server/http/interceptors/interceptors.go

```diff
@@ -37,15 +38,41 @@ import (
 
 var (
 	upgradeInsecure = flag.Bool("ssl.upgrade_insecure", false, "True if http requests should be redirected to https. Assumes http traffic is served on port 80 and https traffic is served on port 443 (typically via an ingress / load balancer).")
-
-	uuidV4Regexp = regexp.MustCompile("[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}")
 )
 
+const ContentSecurityPolicyHeader = "Content-Security-Policy"
+
+func GetContentSecurityPolicy(scriptHashes ...[sha256.Size]byte) string {
+	scriptSrcs := make([]string, 0, len(scriptHashes))
+	for _, scriptHash := range scriptHashes {
+		scriptSrcs = append(scriptSrcs, fmt.Sprintf("'sha256-%s'", base64.StdEncoding.EncodeToString(scriptHash[:])))
+	}
+	return strings.Join([]string{
+		"default-src 'self'",
```

Comment:
I clicked around in the app and it worked, but please double-check this list and try to think of other kinds of access it may need.